addons/web/static/src/views/fields/char/char_field.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 .o_field_input_buttons {
-    position: absolute;
+    position: relative;
```

Comment:
:no_good: :red_square: 
This breaks every other use of the buttons (e.g. translatable chars)

Reply:
You are right, but changing the position-absolute to position-relative did not work.

Reply:
Hello, what do you think of the changes ? Thank you for your help